### DISCUSSION TITLE
PrimeNG v17 components accessibility issue

### DISCUSSION THREAD

**Discussion by impetoyt** [primefaces/community#1473] [PrimeNG] [answered]
I report two issue regard PrimeNG v17 components in the PrimeNG website components section:

1) The MegaMenu component has a problem with the keyboard navigation: after the focus is moved on the first element of a drop-down menu than is not possible to move the focus with the arrows-next (or left) on the next (or previous) element of the menu. Try in the example it will move on the parent elemnt (instead PrimeNG v16 works fine)
2) The Carousel component: once you move the focus inside a carousel page, the user can't go back with TAB key (or any other key) to the indicators button (or arrow indicators) in order to change the carousel page. The TAB key will move the focus to every button of all carousel pages also if the user dosn't change the carousel page yet.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thanks for reporting the issues. We'll address them in upcoming releases.

https://github.com/primefaces/primeng/issues/15111
https://github.com/primefaces/primeng/issues/14833
